Clinical-Genomics/scout
# Validate ClinVar submission refseq:HGVSs using VariantValidator

Pre-validate the refseq:hgvs items shown in the form ClinVar add variant:


<img width="1095" alt="image" src="https://user-images.githubusercontent.com/28093618/200279455-c7075b71-cbc0-442c-8012-3261b9b7d19f.png">

Validation is performed using the [VariantValidator API](https://rest.variantvalidator.org/)



<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>


**How to test**:
1. Locally or on stage, create a ClinVar submission (or add to an existing one) with a SNV variant. Make sure that transcripts for this variant are validated during the submission process

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
